src/types/customerInstitution.types.ts

```diff
@@ -0,0 +1,38 @@
+export interface CustomerInstitution {
+  id: string;
+  name: string;
+  displayName: string;
+  shortName: string;
+  archiveName: string;
+  CNAME: string;
+  institutionDNS: string;
```

Comment:
institutionDns? 
samme over -> "cname"
Men dette er sikkert ting som vil avgjøres av backend uansett..?

Reply:
si det? Disse skrives jo helst med store bokst.

Reply:
hadde samme greia med Api og Url også. Vi bestemte oss vel for å få for camelCase uansett

Reply:
ok da